code/modules/admin/verbs/randomverbs.dm

```diff
@@ -674,7 +674,7 @@ Traitors and the like can also be revived with the previous role mostly intact.
 		return
 	var/atom/A = D
 	var/coords = istype(A) ? "at ([A.x], [A.y], [A.z])" : ""
-	if(alert(src, "Are you sure you want to delete:\n[D]\n[coords]?", "Confirmation", "Yes", "No") == "Yes")
+	if(tgui_alert(src, "Are you sure you want to delete:\n[D]\n[coords]?", "Confirmation", list("Yes", "No")) == "Yes")
```

Comment:
Admin stuff

Reply:
Do you think we should delete it? It's still not a very important thing, and so you can open several alerts and delete them together when you need to

Reply:
Yes. CMD admin delete is critical enough to be a normal alert IMO

Reply:
On TG that's use TGUI Alert, but fine, i'll revert that